src/util/system.cpp

```diff
@@ -366,14 +366,12 @@ bool ArgsManager::ParseParameters(int argc, const char* const argv[], std::strin
     }
 
     // we do not allow -includeconf from command line
-    bool success = true;
     if (auto* includes = util::FindKey(m_settings.command_line_options, "includeconf")) {
-        for (const auto& include : util::SettingsSpan(*includes)) {
```

Comment:
fad0867d6ab9430070aa7d60bf7617a6508e0586: This loop can't be removed, because it implicitly checks whether the span is empty